pkg/rancher-desktop/assets/scripts/spin-operator.namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
As discussed offline, we're going to merge the various spin-kube YAML files to one (with multiple documents) to reduce the number of files we'll need to deal with.